crates/turborepo-ui/src/tui/clipboard.rs

```diff
@@ -0,0 +1,113 @@
+// Inspired by https://github.com/pvolok/mprocs/blob/master/src/clipboard.rs
+use std::process::Stdio;
+
+use base64::Engine;
+use which::which;
+
+pub fn copy_to_clipboard(s: &str) {
+    match copy_impl(s, &PROVIDER) {
+        Ok(()) => (),
+        Err(err) => tracing::debug!("Unable to copy: {}", err.to_string()),
+    }
+}
+
+#[allow(dead_code)]
+enum Provider {
+    OSC52,
+    Exec(&'static str, Vec<&'static str>),
+    #[cfg(windows)]
+    Win,
+    NoOp,
+}
+
+#[cfg(windows)]
```

Comment:
would it make sense to fold these into an if statement with `cfg!`? That way the compiler can check the code paths